src/act_comm.cpp

```diff
@@ -98,9 +98,9 @@ void announce(const char *buf, CHAR_DATA *ch) {
     for (d = descriptor_list; d != nullptr; d = d->next) {
         CHAR_DATA *victim;
 
-        victim = d->original ? d->original : d->character;
+        victim = d->person();
```

Comment:
I had to check what announce() actually did :P

Reply:
Yeah! this has been a great fun trip around the code. I forgot what `yell` did too!